### DISCUSSION TITLE
Table search does't work in windows 7

### DISCUSSION THREAD

**Discussion by your-mirror** [dbeaver/dbeaver#33] [Q&A] [answered]
Doesn't work ctrl+f in windows 7, dbeaver 3.5.2.

In linux version when I press ctrl+f I can see a searching input for a table, but in the Windows it doesn't work.

**Comment by serge-rider** [comment]
Can't reproduce. 
If you are searching in a query results then focus has to be in results grid.

**Comment by your-mirror** [comment]
I do it in the window "Database Navigator" when try to find a table by name...

**Comment by serge-rider** [comment, marked as answer]
This feature (ctrl+f in table/tree control) is a native feature of Linux windows manager. Windows doesn't have it. In Windows you can start typing table name and it will position to an item with matching name.
Also you can open table list (double click on "Tables" folder") and press ctrl+f there.

**Comment by your-mirror** [comment]
Sorry...
